src/aap_eda/settings/default.py

```diff
@@ -448,7 +450,27 @@ def _get_secret_key() -> str:
 # ---------------------------------------------------------
 # RULEBOOK ENGINE LOG LEVEL
 # ---------------------------------------------------------
-ANSIBLE_RULEBOOK_LOG_LEVEL = settings.get("ANSIBLE_RULEBOOK_LOG_LEVEL", "-v")
+
+
+# For backwards compatibility, from the old value "-v" to the new value "info"
```

Comment:
@Alex-Izquierdo Would it handle this old settings
```
EDA_ANSIBLE_RULEBOOK_LOG_LEVEL='@none None'
```
Will the old EDA_ANSIBLE_RULEBOOK_LOG_LEVEL have to be renamed to RULEBOOK_PROCESS_LOG_LEVEL should we be supporting both for backward compatibility

Reply:
Hi @mkanoor None value is already handled by the default value of the `get" method:
https://github.com/ansible/eda-server/pull/685/files#diff-b1dc31d817d52a3e8b384026c1442223022f882ba1898b45998239041d8c3611R459

> Will the old EDA_ANSIBLE_RULEBOOK_LOG_LEVEL have to be renamed to RULEBOOK_PROCESS_LOG_LEVEL should we be supporting both for backward compatibility

Is it a question or afirmation? Why we would change the variable name? ANSIBLE_RULEBOOK_LOG_LEVEL seems to me good enough.

Reply:
The current default **is** EDA_ANSIBLE_RULEBOOK_LOG_LEVEL
```
tools/docker/docker-compose-dev.yaml:
  16    - EDA_SERVER=http://${EDA_HOST:-eda-api:8000}
  17:   - EDA_ANSIBLE_RULEBOOK_LOG_LEVEL=${EDA_ANSIBLE_RULEBOOK_LOG_LEVEL:-'-v'}
  18    - EDA_CONTAINER_NAME_PREFIX=${EDA_CONTAINER_NAME_PREFIX:-eda}

tools/docker/docker-compose-mac.yml:
  17    - EDA_SERVER=http://${EDA_HOST:-eda-api:8000}
  18:   - EDA_ANSIBLE_RULEBOOK_LOG_LEVEL=${EDA_ANSIBLE_RULEBOOK_LOG_LEVEL:-'-v'}
  19    - EDA_CONTAINER_NAME_PREFIX=${EDA_CONTAINER_NAME_PREFIX:-eda}

tools/docker/docker-compose-stage.yaml:
  18    - EDA_SERVER=http://eda-api:8000
  19:   - EDA_ANSIBLE_RULEBOOK_LOG_LEVEL=${EDA_ANSIBLE_RULEBOOK_LOG_LEVEL:-'-v'}
  20    - EDA_CONTAINER_NAME_PREFIX=${EDA_CONTAINER_NAME_PREFIX:-eda}
```

Reply:
Just to remind that "EDA" prefix is automatically added for every exposed variable. So the environment variable for "ANSIBLE_RULEBOOK_LOG_LEVEL" is "EDA_ANSIBLE_RULEBOOK_LOG_LEVEL"

Reply:
I see what is the misunderstading. At some point I have renamed the variable name, but that was never my intention. In my mind was always ANSIBLE_RULEBOOK_LOG_LEVEL, not sure when I changed it. I will fix it.

Reply:
changed.